src/main/java/net/smartcosmos/usermanagement/tenant/resource/CreateTenantResource.java

```diff
@@ -36,7 +37,7 @@
     private CreateTenantService service;
 
     @Autowired
-    public CreateTenantResource(CreateTenantService service) { this.service = service; }
+    public CreateTenantResource(CreateTenantServiceDefault service) { this.service = service; }
```

Comment:
why specifically wire the Default implementation?  Generally you want to wire the interface.